tests/event/test_base_event.py

```diff
@@ -70,8 +70,8 @@ def test_create_base_event(event_id, group_id):
     assert event.dict() == expected
 
 
-@pytest.mark.parametrize('event_id', [None, -1, -1000])
-def test_create_base_event_fail_id(event_id):
+@pytest.mark.parametrize('event_id', [-1, -1000])
```

Comment:
Did you remove the `None` test case?

Reply:
Yes, `BaseEvent.id` is not `Optional` so there's no reason to test `None` here, this test was not necessary. Now that it's typed, mypy complained about using an optional value in a non-optional property.

Reply:
Got it. Thanks!